src/pytest_bdd/parser.py

```diff
@@ -100,6 +101,19 @@ def __bool__(self) -> bool:
         return bool(self.examples)
 
 
+@dataclass(eq=False)
+class Rule:
+    keyword: str
+    name: str
+    description: str
+    tags: set[str]
+    feature: Feature
```

Comment:
Is this really needed?

Reply:
Which part?  The whole dataclass or the feature field?

Reply:
The feature field

Reply:
Well spotted.  I probably used it when I first implemented somewhere, but the field is no longer used anywhere.  Removed.